db/migrations/versions/013_project_table_to_jsonb.py

```diff
@@ -0,0 +1,76 @@
+"""empty message
+
+Revision ID: 013_project_table_to_jsonb
```

Comment:
just make sure you align with Joel on this version number, as I think he has an open branch with migration in too?